api/IDBEnvironment.json

```diff
@@ -0,0 +1,110 @@
+{
+  "api": {
+    "IDBEnvironment": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/IDBEnvironment",
+        "support": {
+          "webview_android": {
+            "version_added": "4.4"
+          },
+          "chrome": [
+            {
+              "version_added": "24"
+            },
+            {
+              "version_added": "23",
+              "version_removed": "57",
+              "prefix": "webkit"
+            }
+          ],
+          "chrome_android": {
+            "version_added": "4.4"
```

Comment:
Chrome Android uses Chromium version numbers, not Android numbers. Please change to `"24"`.

Reply:
👍